client/src/shared/components/Modal/Modal.tsx

```diff
@@ -25,14 +22,14 @@ const Modal: React.FC<ModalProps> = ({
           ></span>
 
           <div
-            className="inline-block align-bottom bg-white rounded-lg text-left overflow-hidden shadow-xl transform transition-all lg:my-8 lg:align-middle w-2/4"
+            className={`inline-block align-bottom bg-white rounded-lg text-left overflow-hidden shadow-xl transform transition-all lg:my-8 lg:align-middle ${
+              className === '' ? 'w-2/4' : className

```

Comment:
You can just add className without condition and pass width to it, so it will ignore the models default width.
Right now if I were to add another className other than width, its gonna remove the models width, which may not be what we want